### ISSUE TITLE
Command "ofxstatement edit-config" fails on Windows, tool.edit_config assumes posix?

### ISSUE THREAD

**Issue by aGrimRepoMan** [kedder/ofxstatement#96]
Hi,

I ran into a traceback when running the "ofxstatement edit-config" command on Windows 10.  I suspect it's because the call to shlex.split() in edit_config() in module tool.py assumes a posix OS?  I have pasted the traceback further below. This was with ofxstatement release 0.6.2.

I was able to resolve the traceback and successfully edit a config file by privately changing line 93 in method edit_config() from:
`subprocess.call(shlex.split(editor) + [configfname])`
to
`subprocess.call(shlex.split(editor, posix=not sys.platform.startswith("win")) + [configfname])`

More details follow.

Expected behaviour:
1. In powershell on Windows 10 PC, run ofxstatement edit-config
2. Edit the config file as desired and save.

Actual behaviour: 
1. In powershell on Windows 10 PC, run ofxstatement edit-config
2. Receive a traceback as follows, with reason "FileNotFoundError: [WinError 2] The system cannot find the file specified"

```
    PS C:\Users\gg> ofxstatement -d edit-config
    INFO: Running editor: C:\Program Files (x86)\Vim\vim81\gvim.exe C:\Users\gg\AppData\Local\ofx\ofxstatement\config.ini
    Traceback (most recent call last):
    File "C:\Users\gg\AppData\Local\Programs\Python\Python38\Scripts\ofxstatement-script.py", line 11, in <module>
       load_entry_point('ofxstatement==0.6.2', 'console_scripts', 'ofxstatement')()
     File "c:\users\gg\appdata\local\programs\python\python38\lib\site-packages\ofxstatement-0.6.2-py3.8.egg\ofxstatement\tool.py", line 155, in run
     File "c:\users\gg\appdata\local\programs\python\python38\lib\site-packages\ofxstatement-0.6.2-py3.8.egg\ofxstatement\tool.py", line 95, in edit_config
     File "C:\Users\gg\AppData\Local\Programs\Python\Python38\lib\subprocess.py", line 340, in call
       with Popen(*popenargs, **kwargs) as p:
     File "C:\Users\gg\AppData\Local\Programs\Python\Python38\lib\subprocess.py", line 854, in __init__
       self._execute_child(args, executable, preexec_fn, close_fds,
     File "C:\Users\gg\AppData\Local\Programs\Python\Python38\lib\subprocess.py", line 1307, in _execute_child
       hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
   FileNotFoundError: [WinError 2] The system cannot find the file specified
```

Actual behaviour after code change:
 - Same as Expected Behaviour

**Comment by kedder** [comment]
@aGrimRepoMan, thanks for digging into it! I haven't tested ofxstatement under windows (and actually never expected it to work), but if this is the only thing that was missing from it being functional, I'll happy to accept this change - it makes total sense.

I will change it to `posix=not os.name not in ['posix', 'mac'])` though. I'm not completely sure about 'mac', I'll have to find someone with modern macbook to test the value of `os.name`. Expect new release soon with the fix.

**Comment by aGrimRepoMan** [comment]
@kedder, Much appreciated.   I haven't had any other problems running on Windows 10.  

Thanks for sharing your work - I wish I'd stumbled on "ofxstatement" earlier!

**Comment by kedder** [comment]
@aGrimRepoMan, ofxstatement 0.6.3 is out, thanks again for the fix and have fun!

**Comment by kedder** [comment]
Unfortunately I made a commit without proper testing and it broke the `edit-config` feature completely. Should be fixed now with 0.6.4.
